.circleci/config.yml

```diff
@@ -266,6 +266,80 @@ jobs:
       - store_artifacts:
           path: apps/mark/integration-testing/cypress/videos/
 
+  # @votingworks/mark-scan-backend
```

Comment:
I don't love this atm because we're re-running the same tests twice, but I feel strongly that mark-scan should be its own app. VxMarkScan will run on a different device and UX will diverge from VxMark; benefits of keeping the apps united are outweighed by how troublesome it would be to maintain 2 separate flows for 2 different devices. As mark-scan diverges running these tests will become worth the time and compute budget to run them. If there's a way to disable them until then while still meeting the requirement that `.circleci/config.yml` is up to date, happy to update.

Reply:
I don't think this is a problem, but if others disagree we can disable the tests by removing (or renaming) the `test` script in `package.json`.